package.json

```diff
@@ -2,6 +2,21 @@
   "name": "aw-client",
   "version": "0.1.0",
   "description": "An ActivityWatch client library for JavaScript/NodeJS",
+  "author": "Johan Bjäreholt <johan@bjareho.lt>",
+  "license": "MPL-2.0",
+  "homepage": "https://github.com/ActivityWatch/aw-client-js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ActivityWatch/aw-client-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ActivityWatch/aw-client-js/issues"
+  },
+  "scripts": {
+    "compile": "tsc",
+    "test": "npm run prepare; ./test.sh",
```

Comment:
What about `npm run prepare && ./test.sh`? In this case it would also fail if the build fails.
(But I don't know how typescript errors count for failing so it would _maybe_ require a clean typescript code)

Reply:
Ah, you're right. Nice catch!